modules/virt-creating-a-service-from-a-virtual-machine.adoc

```diff
@@ -8,6 +8,26 @@
 
 Create a service from a running virtual machine by first creating a `Service` object to expose the virtual machine.
 
+[NOTE]
+====
+If IPv4 and IPv6 dual-stack networking is enabled for your cluster, you can create a service that can use IPv4, IPv6, or both, by defining the `spec.ipFamilyPolicy` and the `spec.ipFamilies` fields in the `Service` object.
```

Comment:
One thing I was not sure about is where these values are specified. I assume that the parameters are in the Service CR manifest (yaml) that is mentioned in step 3. It might be clearer to the user if you were to add the parameters to the yaml and then put this note in a callout. In its current location, the context is not immediately clear.